lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -62,15 +70,53 @@ def diode_params_from_voltage(
     )
 
 
-def viscosity_of_water(temperature):
-    """Computes the viscosity of water in [Pa*s] at a particular temperature.
+def density_of_water(temperature, molarity, pressure=0.101325):
+    """Determine the density of water with NaCl.
 
-    These equations come from section 3.7 from [1]_.
+    This model is based on [1]_.
+
+    Parameters
+    ----------
+    temperature : array_like
+        Temperature in C
+    molarity : float
+        Molarity NaCl [mol/L]
+    pressure : float, optional
+        Pressure (default: 0.101325) [MPa].
+
+    Raises
+    ------
+    ValueError
+        When the provided values are outside the valid range of this model. The valid ranges are:
+        Temperature (20°C <= T < 150°C), pressure <= 35 MPa and molality <= 6 mol/kg.
+
+    References
+    ----------
+    .. [1] Kestin, J., Khalifa, H. E., & Correia, R. J. (1981). Tables of the dynamic and
+       kinematic viscosity of aqueous NaCl solutions in the temperature range 20–150 C and
+       the pressure range 0.1–35 MPa. Journal of physical and chemical reference data, 10(1),
+       71-88.
+    """
+    molality = molarity_to_molality(molarity, temperature=temperature, pressure=pressure)
```

Comment:
how can you call this without the molecular weight argument?

Reply:
Good catch. Turns out it fell outside the test coverage when I reshuffled to use molarity. Fixed now.